osbs/utils/__init__.py

```diff
@@ -588,6 +590,17 @@ def retry(*args, **kwargs):
     return do_retry_on_exception
 
 
+def user_warning_log_handler(self, message):
```

Comment:
So we don't want py typing in new code? (as you didn't transformed it from my example here)

Reply:
Missing docstring

Reply:
Because osbs-client still supports Py2 and function annotations aren't implemented there (see [PEP 3107](https://www.python.org/dev/peps/pep-3107/)), we cannot add it here :(

Reply:
Aah, true sorry. I'm living on the edge